examples/DGmethods/ex_001_dycoms.jl

```diff
@@ -120,53 +120,184 @@ end
+  qt_localtot = sum(localQ[:, 6, :])
+  qliq_localtot = sum(thermoQ[:, 1, :])
+  qice_localtot = sum(thermoQ[:, 2, :])
+  qvap_localtot = sum(thermoQ[:, 3, :])
+  T_localtot = sum(thermoQ[:, 4, :])
+  theta_localtot = sum(thermoQ[:, 5, :])
+  mpirank = MPI.Comm_rank(MPI.COMM_WORLD)
+  nranks = MPI.Comm_size(MPI.COMM_WORLD)
+  rho_tot = MPI.Reduce(rho_localtot, +, 0, MPI.COMM_WORLD)
+  U_tot = MPI.Reduce(U_localtot, +, 0, MPI.COMM_WORLD)
+  V_tot = MPI.Reduce(V_localtot, +, 0, MPI.COMM_WORLD)
+  W_tot = MPI.Reduce(W_localtot, +, 0, MPI.COMM_WORLD)
+  e_tot = MPI.Reduce(e_localtot, +, 0, MPI.COMM_WORLD)
+  qt_tot = MPI.Reduce(qt_localtot, +, 0, MPI.COMM_WORLD)
+  qliq_tot =MPI.Reduce(qliq_localtot, +, 0, MPI.COMM_WORLD)
+  qice_tot =MPI.Reduce(qice_localtot, +, 0, MPI.COMM_WORLD)
+  qvap_tot =MPI.Reduce(qvap_localtot, +, 0, MPI.COMM_WORLD)
+  T_tot =MPI.Reduce(T_localtot, +, 0, MPI.COMM_WORLD)
+  theta_tot =MPI.Reduce(theta_localtot, +, 0, MPI.COMM_WORLD)
+  if mpirank == 0
+    rho_avg = rho_tot / (size(localQ, 1) * size(localQ, 3) * nranks)
+    U_avg = (U_tot / (size(localQ, 1) * size(localQ, 3) * nranks))/rho_avg
+    V_avg = (V_tot / (size(localQ, 1) * size(localQ, 3) * nranks))/rho_avg
+    W_avg = (W_tot / (size(localQ, 1) * size(localQ, 3) * nranks))/rho_avg
+    e_avg = (e_tot / (size(localQ, 1) * size(localQ, 3) * nranks))/rho_avg
+    qt_avg = (qt_tot / (size(localQ, 1) * size(localQ, 3) * nranks))/rho_avg
+    qliq_avg = (qliq_tot / (size(localQ, 1) * size(localQ, 3) * nranks))/rho_avg
+    qice_avg = (qice_tot / (size(localQ, 1) * size(localQ, 3) * nranks))/rho_avg
+    qvap_avg = (qvap_tot / (size(localQ, 1) * size(localQ, 3) * nranks))/rho_avg
+    T_avg = (T_tot / (size(localQ, 1) * size(localQ, 3) * nranks))
+    theta_avg = (theta_tot / (size(localQ, 1) * size(localQ, 3) * nranks))
+    @info "ρ average = $(rho_avg)"
+    @info "U average = $(U_avg)"
+    @info "V average = $(V_avg)"
+    @info "W average = $(W_avg)"
+    @info "e average = $(e_avg)"
+    @info "qt average = $(qt_avg)"
+    @info "qliq average = $(qliq_avg)"
+    @info "qice average = $(qice_avg)"
+    @info "qvap average = $(qvap_avg)"
+    @info "T average = $(T_avg)"
+    @info "theta average = $(theta_avg)"
   end
-
-  # standard_deviation and variance
-  rho_local_flucttot = sum(varia[:,1,:])
+  AVG=SVector(rho_avg,U_avg,V_avg,W_avg,e_avg,qt_avg)
+  AVG_T=SVector(qliq_avg,qice_avg,qvap_avg,T_avg,theta_avg)
+  #fluctuations
+  for s in 1:6	
+	for e in 1:nrealelems
+		for i in 1:Nq*Nqk*Nq
+			if s==1
+				fluctQ[i,s,e] = localQ[i,s,e]-AVG[s]
+				VarQ[i,s,e]=fluctQ[i,s,e]^2
+				fluctT[i,s,e] = thermoQ[i,s,e]/localQ[i,s,e]-AVG_T[s]
+			elseif s<6
+				fluctQ[i,s,e] = localQ[i,s,e]/localQ[i,1,e]-AVG[s]
+                                VarQ[i,s,e]=fluctQ[i,s,e]^2
+				fluctT[i,s,e] = thermoQ[i,s,e]/localQ[i,s,e]-AVG_T[s]
+			else
+				fluctQ[i,s,e] = localQ[i,s,e]/localQ[i,1,e]-AVG[s]
+                                VarQ[i,s,e]=fluctQ[i,s,e]^2
```

Comment:
To compute the variances after accumulating time averages, you can also compute them as `(localQ[i,s,e]/localQ[i,1,e])^2 - AVG[s]^2`